libs/header/interface.go

```diff
@@ -8,6 +8,11 @@ import (
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 )
 
+const (
+	// MaxRangeRequestSize defines the max amount of headers that can be handled/requested at once.
+	MaxRangeRequestSize uint64 = 512
+)
+
```

Comment:
I am not opposed to this one being public and want to understand why only this is part of the public interface and all other changed params are private.

Reply:
Which one exactly? Other options are related to the peerTracker and should be hidden from the user.

```
const (
 	// defaultScore specifies the score for newly connected peers.
 	defaultScore float32 = 1
 	// maxTrackerSize specifies the max amount of peers that can be added to the peerTracker.
 	maxPeerTrackerSize = 100
 )

 var (
 	// maxAwaitingTime specifies the duration that gives to the disconnected peer to be back online,
 	// otherwise it will be removed on the next GC cycle.
 	maxAwaitingTime = time.Hour
 	// gcCycle defines the duration after which the peerTracker starts removing peers.
 	gcCycle = time.Minute * 30
 )
```

Reply:
The second argument is that all these options are used in one place(closely to the peerTracker). But we are using `MaxRangeRequestSize` in two different packages.

Reply:
> But we are using MaxRangeRequestSize in two different packages.
ah, right ok